microcosm_flask/swagger/schemas.py

```diff
@@ -65,9 +65,15 @@ def iter_fields(self, schema: Schema) -> Iterable[Tuple[str, Field]]:
         Generates: name, field pairs
 
         """
+        serialized_names = {}
+        for field_name, field_def in schema.declared_fields.items():
+            if isinstance(field_def, Field) and field_def.data_key:
+                serialized_names[field_name] = field_def.data_key
+
         for name in sorted(schema.fields.keys()):
```

Comment:
It seems weird to be iterating once over `schema.declared_fields` and once over `schema.fields`. Seems like we could have a single loop.

Reply:
Removing the loop and the tests pass fine
```
diff --git a/microcosm_flask/swagger/schemas.py b/microcosm_flask/swagger/schemas.py
index d1a8d87..1bd27a9 100644
--- a/microcosm_flask/swagger/schemas.py
+++ b/microcosm_flask/swagger/schemas.py
@@ -65,15 +65,11 @@ class Schemas:
         Generates: name, field pairs

         """
-        serialized_names = {}
-        for field_name, field_def in schema.declared_fields.items():
-            if isinstance(field_def, Field) and field_def.data_key:
-                serialized_names[field_name] = field_def.data_key
-
         for name in sorted(schema.fields.keys()):
             field = schema.fields[name]
-            serialized_name = serialized_names.get(name, name)
-            yield serialized_name, field
+            if isinstance(field, Field) and field.data_key:
+                name = field.data_key
+            yield name, field

     def iter_schemas(self, schema: Schema) -> Iterable[Tuple[str, Any]]:
         """
```

Is there some other case you a guarding against with the separate loop @GabeLambda ?

Reply:
From memory, there was some discrepancy I had found between fields and declared_fields (one had `data_key`, the other didn't). But apparently I was mistaken.